src/shared-components/SearchBar/SearchBar.module.css

```
@@ -1,28 +1,31 @@
   transition:
-    border-color 0.2s ease,
-    box-shadow 0.2s ease;
+    border-color var(--transition-duration-fast) ease,
+    box-shadow var(--transition-duration-fast) ease;
   overflow: hidden;
   flex-grow: 1;
   flex-shrink: 1;
   width: auto;
-  min-width: var(--space-12);
+  min-width: var(--space-30);
   max-width: 100%;
```

Stylelint (17.3.0)

[error] Unexpected duplicate "max-width" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)